_harp/css/components/sections/_main-preview.scss

```diff
@@ -21,6 +22,7 @@
 #preview-player {
   margin: 0 auto 30px;
   width: 100%;
+  padding-top: $preview-video-ratio * 100%;
```

Comment:
My latest commit (https://github.com/videojs/videojs.com/pull/49/commits/136f4cba818f8354a3f2a48337ccf62104b0c95e) uses a placeholder class that gets removed when the player has loaded.